pallets/order-book/src/lib.rs

```diff
@@ -32,6 +32,9 @@
 #![allow(dead_code)] // todo (m.tagirov) remove
 // TODO #167: fix clippy warnings
 #![allow(clippy::all)]
+// too many benchmarks, doesn't compile otherwise
+#![recursion_limit = "512"]
+#![feature(int_roundings)]
```

Comment:
What if we move it in `limit_order.rs`, because I don't see how it is used in the current file

Reply:
Is it available with nightly only?

Reply:
1. It's a crate attribute, so it must be placed in the crate root file
2. It was stabilized yesterday in Rust 1.73. However, we use nightly (2023-03-21?), thus I'm enabling the feature. Btw it seems to work on stable 1.73+ as well (by ignoring the attribute, I suppose).